portality/templates/includes/header-secondary-navigation.html

```diff
@@ -6,10 +6,10 @@
 {% endif %}
 
 <li class="dropdown {{ extraClasses }}">
-    <span class="dropdown__trigger nav-bar__link" data-toggle="dropdown" id="menu-{{ selected_nav.id }}" role="button" aria-haspopup="true" aria-expanded="false">
+    <a href="" class="dropdown__trigger nav-bar__link" data-toggle="dropdown" id="menu-{{ selected_nav.id }}" role="button" aria-haspopup="true" aria-expanded="false">
```

Comment:
We shouldn’t be using anchors that aren’t a link / don’t have an `href`. Another way to do it semantically and accessibly could be to use the `<button>` element, though some style changes would be required so that it doesn’t get displayed as a UI button (i.e. only elements using `.button` should be styled, not the HTML element). This way, you can tab through all menu items in sequential order.

Reply:
Also just remembered that I had changed the anchor to a `<span>` to avoid the empty `href` originally. 
 
This is because `<a href="...">` should be taking the user to another page / elsewhere whereas `<button>s` indicates an action. 

It’s a bit complicated though because this is still very debatable. An anchor with an empty `href` is still valid HTML but not necessarily correct. 

The `<button>` article on MDN also states:
> If your buttons are not for submitting form data to a server, **be sure to set their type attribute to button**. Otherwise they will try to submit form data and to load the (nonexistent) response, possibly destroying the current state of the document.

We can discuss this on Monday if you’d like so that we can simply agree on a common (accessible) approach going forward .

Reply:
I did not plan to release it before talking to you. I just tried to find the way to avoid tabindex and the anchor is the way that you did this in the primary nav. But that is why I asked you to talk to you on Monday - cause I am not a big fun of that solution as well.

Reply:
@sssoz I think we need to tackle the issue now. 

It looks like w3 actually accepts anchors with empty href as a good pattern for that use case and operates with `tabindex` property.

https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html